pypiper/manager.py

```diff
@@ -146,6 +146,7 @@ def __init__(
         # this could become customizable if necessary
         self.mem = params['mem'] + "m"
         self.container = None
+        self.clean_initialized = False
```

Comment:
It looks like there may not be a hook to set this; is that true? Is it more experimental for now, or otherwise intended that way?

Reply:
Hahaha yeah I realize now that I didn't exactly think this one through. You're right, no need for parameter. I guess it could be "protected" but either way seems fine.